pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -948,17 +938,19 @@ func (sqlPlugin) GetPluginInfo(context.Context, *spi.GetPluginInfoRequest) (*spi
 	return &pluginInfo, nil
 }
 
-// listMatchingEntries finds registered entries containing all specified selectors. Note
-// that entries containing _more_ than the specified selectors may be returned, since
-// that is also considered a "match"
+// listMatchingEntries finds registered entries containing all specified selectors.
```

Comment:
nit: something like `containing exactly the specified selectors` will help to indicate that this function won't return entries with more than just the specified selectors

Reply:
i like that. will change.